### PR TITLE
HTML FORM

### PR DESCRIPTION
**Pull request by mujohn26**
#### What does this PR do?
- Html input form
#### Description of Task to be completed?
- The added header of the page
-  Added page icons
- added form to the page
-  added screenshot it README file
#### How should this be manually tested?
- clone the repository
- open index.html in your browser

#### Any background context you want to provide?
- N/A

#### Screenshots (if appropriate)
Screen Shot 2020-05-06 at 15 53 58Screen Shot 2020-05-06 at 11 30 27

#### Questions:
- N/A

### CODE REVIEW

**Comment by santiago-dev96** [comment]
## Pull Request under review by @santiago-rodrig (TSE)

File: css/index.css [hunk 1]
```diff
@@ -74,7 +73,100 @@ body {
...
 }
 
-.form-div {
+.mainPage {
+  height: 600px;
+  width: 380px;
+  background-color: #fff;
+  border: 1px solid #c7c7c7;
+  margin: 30px 200px 50px 520px;
+  position: center;
```
Comment (santiago-dev96) [inline css/index.css]: The `position` property is not being properly used, you should read about the property [here](https://developer.mozilla.org/en-US/docs/Web/CSS/position).
```suggestion
  margin: 30px auto 50px auto;
```

File: index.html [hunk 2]
```diff
@@ -27,42 +23,79 @@
...
             </div>
             <div class="sub-menu-icon">
-                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" /></a></div>
+                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" alt="" /></a></div>
                 &nbsp;
                 <div> <a href="#">quickbooks</a></div>
             </div>
             <div class="sub-menu-icon">
-                <div><a href="#"> <img src="./images/icon3.svg" width="23" height="23" /></a></div>
+                <div><a href="#"> <img src="./images/icon3.svg" width="23" height="23" alt=""/></a></div>
```
Comment (santiago-dev96) [inline index.html]: I think the alt should have some content; if it doesn't have any then it is useless for the visually impaired people. Also, your icons are untidy.
```suggestion
                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" alt="turbotax logo"/></a></div>
```

File: index.html [hunk 3]
```diff
@@ -27,42 +23,79 @@
...
         <div class="main-icon">
             <div class="sub-menu-icon">
-                <div><a href="#"><img src="./images/icon1.svg" width="23" height="23" /></a> </div>
+                <div><a href="#"><img src="./images/icon1.svg" width="23" height="23" alt="" /></a> </div>
                 &nbsp;
                 <div> <a href="#">mint</a></div>
             </div>
             <div class="sub-menu-icon">
-                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" /></a></div>
+                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" alt="" /></a></div>
```
Comment (santiago-dev96) [inline index.html]:
```suggestion
                <div><a href="#"> <img src="./images/icon1.svg" width="23" height="23" alt="quickbooks logo" /></a></div>
```

File: index.html [hunk 4]
```diff
@@ -27,42 +23,79 @@
     </div>
     <div class="main-body">
         <div class="main-logo">
-            <img src="./images/icon4.svg" width="218" ; height="55" ; />
+            <img src="./images/icon4.svg" width="218"  height="55" alt="" />
         </div>
         <div class="main-icon">
             <div class="sub-menu-icon">
-                <div><a href="#"><img src="./images/icon1.svg" width="23" height="23" /></a> </div>
+                <div><a href="#"><img src="./images/icon1.svg" width="23" height="23" alt="" /></a> </div>
```
Comment (santiago-dev96) [inline index.html]:
```suggestion
                <div><a href="#"><img src="./images/icon3.svg" width="23" height="23" alt="mint logo" /></a> </div>
```

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required :no_good_man: 
1. Your icons are not in the correct order (yes, it is a requirement)

image

2. Your form is not centered, I suggested some changes regarding that.

Please submit another CRR (Code Review Request) after implementing the changes. Have a nice day :wave:

**Comment by mujohn26** [comment]
Hello @santiago-rodrig  thanks for the feedback and it was resolved.

**Review by serdg0 — CHANGES_REQUESTED** [review]
Hello @mujohn26 !

All the changes are good but still there is an issue. The intuit logo in the top of the form is not properly centered. I see you used a margin-left to try to center it, but I don't recommend it. I'm attaching the image.

Selection_009

- [ ] Center main-logo. **HINT**: Use flex!.

**Happy Coding!** :computer: :nerd_face:

**Comment by mujohn26** [comment]
> Hello @mujohn26 !
> 
> All the changes are good but still there is an issue. The intuit logo in the top of the form is not properly centered. I see you used a margin-left to try to center it, but I don't recommend it. I'm attaching the image.
> 
> Selection_009
> 
> * [ ]  Center main-logo. **HINT**: Use flex!.
> 
> **Happy Coding!** 💻 🤓


Hello, @serdg0 Thanks for the feedback was really very helpful and it was resolved.

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

Great job on implementing the changes requested!

You can go ahead and merge this pull request.

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**